packages/hooks/src/hooks/useTokenBalance/ERC20.json

```diff
@@ -0,0 +1,222 @@
+[
```

Comment:
Did you write this, or was it generated/sourced from elsewhere? I wonder if we can import this from a library instead of needing to define the schema for an ERC20 contract

Reply:
i copied it from the following  [sitw](https://ethereumdev.io/abi-for-erc20-contract-on-ethereum/) but I've you know a library that provides the interface we should replace the .json file

Reply:
I looked around, but couldn't find a legit looking library for this anywhere (i found one with only 1 star, but that seems risky to rely on).

Maybe let's rename the file to `ERC20ABI.json` at least, for a bit more clarity? I assume it'll never need to change (i'm still a bit new to web3, so don't have a full understanding of this yet)

Reply:
Yes the ERC20 interface will not change over time. Will rename the file to achieve more clearity